R/mobr.R

```diff
@@ -10,6 +10,9 @@
 #'   column(s) has to have names "x" and/or "y".
 #' @param binary whether the plot by species matrix "comm" is in abundances or 
 #'   presence/absence.
+#' @param latlong whether the coordinates are in latitude-longitude. If TRUE, it
+#'   is assumed that column "x" is longitude and column "y" is latitude, both in 
+#'   radians.
```

Comment:
I think it's an easy conversion. If you'd like me to implement it just let me know, or feel free to change it.